crates/optimism/node/tests/e2e/p2p.rs

```diff
@@ -43,13 +43,19 @@ async fn can_sync() -> eyre::Result<()> {
         .update_optimistic_forkchoice(canonical_chain[tip_index - reorg_depth])
         .await?;
     second_node
-        .wait_block((tip - reorg_depth) as u64, canonical_chain[tip_index - reorg_depth], true)
+        .wait_block((tip - reorg_depth) as u64, canonical_chain[tip_index - reorg_depth], false)
```

Comment:
this marks whether we need to wait for `Finish` stage to advance which might not happen if persistence threshold is not reached